test/skimNtuple.cpp

```diff
@@ -1017,8 +1034,11 @@ int main (int argc, char** argv)
   for (int i = 0 ; i < 2; i++)
         myIDandISOScaleFactor[i] = new ScaleFactor();
 
-  myIDandISOScaleFactor[0] -> init_ScaleFactor("weights/HTT_SF_2016/Muon/Run2017/Muon_IdIso_IsoLt0.15_eff_RerecoFall17.root");
-  myIDandISOScaleFactor[1] -> init_ScaleFactor("weights/HTT_SF_2016/Electron/Run2017/Electron_IdIso_IsoLt0.10_eff_RerecoFall17.root");
+  //myIDandISOScaleFactor[0] -> init_ScaleFactor("weights/HTT_SF_2016/Muon/Run2017/Muon_IdIso_IsoLt0.15_eff_RerecoFall17.root");
+  //myIDandISOScaleFactor[1] -> init_ScaleFactor("weights/HTT_SF_2016/Electron/Run2017/Electron_IdIso_IsoLt0.10_eff_RerecoFall17.root");
+
+  myIDandISOScaleFactor[0] -> init_ScaleFactor("weights/idIso_SF_2018/Muon_Run2018_IdIso.root");
```

Comment:
move this weight file to "/weights/HTT_SF_2016/Muon/Run2018/" there is no need to create a new "idIso_SF_2018" folder